packages/sdk/src/utils/index.ts

```diff
@@ -38,8 +38,34 @@ export function createTransaction(
   return bitcoin.payments[type]({ pubkey: key, network: networkObj });
 }
 
-export function hdNodeToChild(node: BIP32Interface, formatType: AddressFormats = "legacy", index = 0) {
-  const fullDerivationPath = DERIVATION_PATHS_WITHOUT_INDEX[formatType] + index;
+export function getDerivationPath(formatType: AddressFormats, accountIndex = 0, index = 0) {
+  const pathFormat = {
+    legacy: `m/44'/0'/${accountIndex}'/0/${index}`,
+    segwit: `m/49'/0'/${accountIndex}'/0/${index}`,
+    bech32: `m/84'/0'/${accountIndex}'/0/${index}`,
+    taproot: `m/86'/0'/${accountIndex}'/0/${index}`
+  };
+  return pathFormat[formatType];
+}
+
+export function getPathLevels(fullDerivationPath: string) {
+  const [, purpose, coinType, account, change, addressIndex] = fullDerivationPath.split("/");
+  return {
+    purpose: purpose.replace("'", ""),
+    coinType: coinType.replace("'", ""),
+    account: account.replace("'", ""),
+    change,
+    addressIndex
+  };
+}
+
+export function hdNodeToChild(
+  node: BIP32Interface,
+  formatType: AddressFormats = "legacy",
+  index = 0,
+  accountIndex = 0
```

Comment:
rename properties as per [BIP44 specification](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels)

```suggestion
  addressIndex = 0,
  account = 0
```